test/debug/info_test.rb

```diff
@@ -49,14 +49,32 @@ def test_info_prints_locals_by_default
         type 'info'
         assert_line_text(
           "%self = main\r\n" \
-          "%return = 11\r\n" \
+          "_return = 11\r\n" \
           "a = 1\r\n" \
           "@var = 10\r\n"
         )
         type 'q!'
       end
     end
 
+    def test_info_conflict_lvar
+      code = <<~RUBY
+      1| def foo _return
+      2|   :ok
+      3| end
+      4| foo :ret
+      RUBY
+
+      debug_code code do
+        type 'b 3'
+        type 'c'
+        assert_line_num 3
+        type 'i'
+        assert_line_text [/#return = :ok/, /_return = :ret/]
```

Comment:
I thought you meant `%` instead of `#` in the description? I'm fine with either though.